charts/kube-master/templates/api.yaml

```diff
@@ -356,6 +358,15 @@ spec:
             - mountPath: /etc/kubernetes/config
               name: wormhole-config
               readOnly: true
+          livenessProbe:
+            exec:
+              command:
+              - sh
+              - -c
+              - |
+                cmp -s /etc/kubernetes/certs/tls-wormhole.pem /tmp/tls-wormhole.livenessprobe
+            initialDelaySeconds: 60
+            periodSeconds: 60
```

Comment:
we should set `timeoutSeconds: 3` to avoid the ominous timeouts we are seeing with the default setting of `1` in the last few months

Reply:
And do we even need a shell invocation here or can we execute `cmp` directly? I think the original needed variable expansion which we don't do here

Reply:
thanks @databus23 - I have added the increased `timeoutSeconds`. 
Tested executing `cmp` directly in qa labs - works fine, too.